test/plain-list/AutosuggestApp.test.js

```diff
@@ -578,13 +578,25 @@ describe('Default Autosuggest', () => {
       shouldRenderSuggestions.resetHistory();
     });
 
-    it('should be called with the right parameters', () => {
+    it('should be called with the right parameters during input', () => {
       focusAndSetInputValue('e');
-      expect(shouldRenderSuggestions.callCount).to.equal(4);
+      blurInput();
+      expect(shouldRenderSuggestions.callCount).to.equal(5);
       expect(shouldRenderSuggestions.getCall(0).args).to.deep.equal(['', 'input-focused']);
       expect(shouldRenderSuggestions.getCall(1).args).to.deep.equal(['e', 'input-changed']);
       expect(shouldRenderSuggestions.getCall(2).args).to.deep.equal(['e', 'suggestions-updated']);
       expect(shouldRenderSuggestions.getCall(3).args).to.deep.equal(['e', 'render']);
+      expect(shouldRenderSuggestions.getCall(4).args).to.deep.equal(['e', 'input-blurred']);
```

Comment:
nit: I'd assert after each interaction instead of asserting everything at once in the end.

Reply:
:+1: done